dashboard/dashboard/templates/racetrack/administration.html

```diff
@@ -66,13 +71,19 @@ <h5 class="card-title">Plugins</h5>
                 <ul>
                 {% for plugin in plugins %}
                     <li>
-                        <span>{{ plugin.name }} (version {{ plugin.version }})</span>
-                        {% if plugin.url %}
-                        <span> - <a href="{{ plugin.url }}" target="_blank">{{ plugin.url }}</a></span>
-                        {% endif %}
-                        {% if user.is_staff %}
-                        <a href="{% url 'dashboard:plugin_config_editor' plugin.name plugin.version %}" class="btn btn-primary align-self-start">Edit Config</a>
-                        <button type="button" class="btn btn-danger button-delete-plugin" plugin-name="{{ plugin.name }}" plugin-version="{{ plugin.version }}">Delete</button>
+                        <div class="row py-1">
+                            <div class="col-md-9">
+                                <span>{{ plugin.name }} (version {{ plugin.version }})</span>
+                            {% if plugin.url %}
+                                <span> - <a href="{{ plugin.url }}" target="_blank">{{ plugin.url }}</a></span>
+                            </div>
+                            {% endif %}
+                            {% if user.is_staff %}
+                            <div class="col-md-3 text-end">
+                                <a href="{% url 'dashboard:plugin_config_editor' plugin.name plugin.version %}" class="btn btn-primary">Edit Config</a>
+                                <button type="button" class="btn btn-danger button-delete-plugin" plugin-name="{{ plugin.name }}" plugin-version="{{ plugin.version }}">Delete</button>
+                            </div>
+                        </div>
```

Comment:
I think that `{% endif %}` below should be one line higher, before enclosing `</div>`, to align it with the other side of "if". If user is not staff, the HTML will be broken otherwise.